### PR TITLE
Bugs/en 2588 fix direct sender

### PR DESCRIPTION
**Pull request by iulianpascalau**

### CODE REVIEW

File: p2p/libp2p/directSender.go [hunk 1]
```diff
@@ -161,27 +160,33 @@ func (ds *directSender) Send(topic string, buff []byte, peer p2p.PeerID) error {
...
 
 	return nil
 }
 
+// SendAsync will asynchronously send a direct message to the connected peer
+func (ds *directSender) SendAsync(topic string, buff []byte, peer p2p.PeerID) {
+	go func() {
+		err := ds.Send(topic, buff, peer)
+		if err != nil {
+			log.Debug(err.Error())
```
Comment (SebastianMarian) [inline p2p/libp2p/directSender.go]: I would change this to log.Info to see if we still have some errors here

Comment (iulianpascalau) [reply p2p/libp2p/directSender.go]: changed to log.Info

File: p2p/libp2p/netMessenger.go [hunk 2]
```diff
@@ -452,7 +452,9 @@ func (netMes *networkMessenger) UnregisterMessageProcessor(topic string) error {
 
 // SendToConnectedPeer sends a direct message to a connected peer
 func (netMes *networkMessenger) SendToConnectedPeer(topic string, buff []byte, peerID p2p.PeerID) error {
-	return netMes.ds.Send(topic, buff, peerID)
+	netMes.ds.SendAsync(topic, buff, peerID)
+
+	return nil
```
Comment (raduchis) [inline p2p/libp2p/netMessenger.go]: Do we still need the error?

Comment (iulianpascalau) [reply p2p/libp2p/netMessenger.go]: Not quite. It just satisfy the interface.

**Pull request merged**